frontend/packages/console-dynamic-plugin-sdk/scripts/generate-doc.ts

```diff
@@ -34,7 +34,13 @@ console.log('Generating Console plugin documentation');
 renderTemplate('scripts/templates/console-extensions.md.ejs', {
   extensions: getConsoleExtensions()
     // Sort extensions by their `type` value
-    .sort((a, b) => a.type.localeCompare(b.type))
+    .sort((a, b) => {
+      if (a.isDeprecated !== b.isDeprecated) {
+        if (a.isDeprecated) return 1;
+        if (b.isDeprecated) return -1;
```

Comment:
Too bad JS booleans don't have an integer-based `compare` function.

I think we can simplify the above two statements with
```ts
return a.isDeprecated ? 1 : -1;
```

Reply:
This is fair... based on the previous statement. Good call, wasn't happy with the code because it then had a fall-through if something weird happened (which it really can't based on how the logic is written lol)